Lib/test/test_zipfile/_path/test_path.py

```diff
@@ -5,7 +5,6 @@
 import pickle
 import stat
 import sys
-import time
```

Comment:
This change will get overwritten unless it's ported to zipp. I'm slightly surprised that there are unused imports, given that I run ruff over the codebase. Oh, I see now. gh-123424 has not yet been backported to zipp, and that's where time was used. So shouldn't be a problem getting this synced once that one is.